static/templates/admin/pagedDisplayForm.html

```diff
@@ -0,0 +1,20 @@
+<iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
+<form id="edit_view_settings" class="edit_form edit_view_settings_form">
+    <div class="form_field">
+        <label for="view_settings_change">Page Display:</label>
+        <select id="view_settings_change">
```

Comment:
I think viewBehavior and direction should be separate inputs since they are independent. So we should have a view behavior input with values "individual" and "paged" (and maybe "continous"). Direction doesn't necessarily have to be addressed in this ticket, I didn't include it in any of the currently created tickets since it was lower priority. I'm pretty sure direction affects "individual" display in UV as well though.